html/siptest.js

```diff
@@ -1519,3 +1541,33 @@ function removeHelper(helperId) {
 		$('#sipcall'+helperId).remove();
 	}
 }
+/**
+ * send message function
+ *
+ * @param {number} suffix helper id
+ */
+function doMessage(suffix) {
```

Comment:
Doxygen style documentation for JavaScript functions is not needed, a brief one-liner comment is more than enough.

Reply:
will do